pom.xml

```
@@ -87,6 +87,7 @@
       <modular.jdk.surefire.arg>--add-modules java.sql,jdk.unsupported </modular.jdk.surefire.arg>
 
       <activemq-artemis-native-version>2.0.0</activemq-artemis-native-version>
+      <artemis-console-war-version>1.1.0</artemis-console-war-version>
```

artemis-console-war-version is used for apache-artemis-console and artemis-console-war, should we remove the war token?
```suggestion
      <artemis-console-version>1.1.0</artemis-console-version>
```

---

Agreed, I made that change with some other fixups in my recent rebase+push.